src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,240 @@
+    </li>
+    <ul>
+      <li>
+        SKY UX demos are authored with <a href="https://github.com/blackbaud/skyux-sdk-cli">SKY UX SDK CLI</a>. If this is your first time using SKY UX Builder, you must install the CLI as a global NPM package before you run <stache-code>skyux serve</stache-code>:
+      </li>
+      <stache-code>npm i -g @skyux-sdk/cli</stache-code>
+    </ul>
+    <li>
+      When you save code changes, the code compiles, static code analysis is performed, unit tests run, and the page refreshes with the latest changes.
+    </li>
+    <p>
+      Keep in mind that when you create components, you must add public exports through <stache-code>./src/core.ts</stache-code> and <stache-code>./src/demo.ts</stache-code> to allow users to import your component modules. The public exports provide entry points for the bundles that SKY UX deploys.
+    </p>
+    <p>
+      The recommended change detection strategy for SKY UX components is to use <a href="https://angular.io/api/core/ChangeDetectionStrategy">the <stache-code>OnPush</stache-code> property instead of <stache-code>Default</stache-code></a>.
+    </p>
+  </ol>
+
+  <stache-page-anchor>
+    Write documentation
+  </stache-page-anchor>
+
+  <p>
+    We recommend that you create reference documentation for the <a stacheRouterLink="/components">Components section</a> of the SKY UX website. Component documentation should be extensive enough to explain the features of your components, including all properties and events. In the <a href="https://github.com/blackbaud/skyux2-docs">SKY UX docs repo</a>, you create subfolders for your components in the <stache-code>src/app/components</stache-code> folder and document them in index.html files. To create action buttons for the Components section landing page, you add entries in the <stache-code>demo-components.service.ts</stache-code> file.
+  </p> 
+  
+  <p>
+    If you do not document your components, the SKY UX team will create the documentation, but we request that you <a href="https://github.com/blackbaud/skyux2-docs/issues">file an issue</a>. However, we strongly encourage you to create your own documentation because it can be useful to uncover issues with your components.
+  </p>
+
+  <stache-page-anchor>
+    Write unit tests
+  </stache-page-anchor>
+
+  <p>
+    Your unit test files should end in <stache-code>.spec.ts</stache-code> and reside in the same folders as the components that they test. For example, the test file for <stache-code>foo/foo.component.ts</stache-code> should be <stache-code>foo/foo.component.spec.ts</stache-code>. All test fixtures such as sample components, mock services, etc. should be in a fixtures folder within the component's folder. The existing codebase contains many examples of this pattern, so just follow the patterns in the existing code when you write tests.
```

Comment:
Should we point out that the command to run unit tests for a library is `skyux test --coverage library`?